### ISSUE TITLE
Document that temporal values can now be stored in variables

### ISSUE THREAD

**Issue by korthout** [camunda-cloud/camunda-cloud-documentation#650]
**Current situation**
https://docs.camunda.io/docs/components/concepts/expressions/#temporal-expressions
>The current date and date-time can be accessed using the built-in functions today() and now(). To store the current date or date-time in a variable, convert it to a string using the built-in function string().

This is no longer true from 1.4 onwards, due to https://github.com/camunda-cloud/zeebe/pull/8842.

**Wanted situation**
We should make sure the documentation is clear about what happens to the result of temporal expressions. This includes checking the documentation for other places where similar things are stated.

**Comment by akeller** [comment]
@korthout is this something @christinaausley could partner with you on to complete?

**Comment by korthout** [comment]
Thanks for pointing it out @akeller 👍 

IMO, this is something the team (zeebe process automation) should plan to work on. I've added it to our project board so that it will be part of our upcoming planning session.

❓ @christinaausley is there anything specific you'd like to see in these docs? If not, I propose that the team follows the regular process: write the documentation and then ask for your review.

**Comment by christinaausley** [comment]
@korthout Agreed here. Loop me in where I can review, but in the meantime I'm also happy to assist with skimming the docs to locate relevant scenarios where it may be unclear what happens to the result of temporal expressions.

**Comment by korthout** [comment]
We want to work on this soon because the feature is already available.

**Comment by saig0** [comment]
Please note that we have an open issue that is related to storing temporal variables: https://github.com/camunda/zeebe/issues/10307. We may want to highlight this case in the docs.

**Comment by megglos** [comment]
Decreasing priority to later, in the process of revising all issues flagged as upcoming.
